docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
     environment:
       DATABASE_URL: postgres://postgres@db/askdarcel_development
       TEST_DATABASE_URL: postgres://postgres@db/askdarcel_test
+      ALGOLIA_APPLICATION_ID:
+      ALGOLIA_API_KEY:
```

Comment:
Or alternatively, could we fallback to a naive search if the credentials aren't available? This would also be useful for developing without an available internet connection and for the testing environment, which probably shouldn't be making real requests to the internet.